cfme/tests/ssui/test_ssui_myservice.py

```diff
@@ -70,6 +70,7 @@ def _finalize():
 
 
 @pytest.mark.rhv3
+@pytest.mark.meta(blockers=[GH('ManageIQ/integration_tests:8200')])
```

Comment:
@jan-zmeskal, please explain why this is bloker. 
As far as I understand you can always move necessary registration statement to one of always imported files.
Besides that you can always add an import statement here as a workaround.